sql/moz-fx-data-shared-prod/firefox_ios/retention_profiles/view.sql

```diff
@@ -0,0 +1,84 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_ios.retention_profiles`
```

Comment:
to align with desktop and the terminology of "Client level DAU", we can call this view retention_client